formats/hocon/src/main/kotlin/kotlinx/serialization/hocon/serializers/JDurationSerializer.kt

```diff
@@ -0,0 +1,56 @@
+package kotlinx.serialization.hocon.serializers
+
+import com.typesafe.config.Config
+import java.time.Duration as JDuration
+import kotlin.time.*
+import kotlinx.serialization.*
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.descriptors.*
+import kotlinx.serialization.encoding.*
+import kotlinx.serialization.hocon.*
+import kotlinx.serialization.hocon.internal.*
+
+/**
+ * Serializer for [JDuration].
+ * For decode using [Duration format][https://github.com/lightbend/config/blob/main/HOCON.md#duration-format].
```

Comment:
Same here: hard to understand. You may want to rephrase this as 'All possible Hocon duration formats [link] are accepted for decoding. During encoding, the serializer emits values using time unit short names: ..... The largest integer time unit is encoded.'

Reply:
Fixed